pennylane/operation.py

```diff
@@ -1727,3 +1727,43 @@ def heisenberg_obs(self, wires):
         p = self.parameters
         U = self._heisenberg_rep(p)  # pylint: disable=assignment-from-none
         return self.heisenberg_expand(U, wires)
+
+
+def operation_derivative(operation) -> np.ndarray:
```

Comment:
Do we want this to be visible in the docs? (it currently would be)

Reply:
I think it's a handy enough function that people might want to know about it.

Reply:
I think it's quite a handy function as well to know about. :slightly_smiling_face:

Reply:
Great, ideally this could change to a function that returns an operator rather than its matrix representation.